src/common/meta/src/rpc/ddl.rs

```diff
@@ -853,6 +856,43 @@ impl From<DropFlowTask> for PbDropFlowTask {
     }
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct QueryContext {
```

Comment:
Why not reuse the existing `QueryContext`?

Reply:
The existing `QueryContext` isn't serializable.